visualization/app/codeCharta/ui/matchingFilesCounter/matchingFilesCounter.component.html

```diff
@@ -1,3 +1,7 @@
-<span class="box-rounded node-count">{{ $ctrl._viewModel.fileCount }}<i class="fa fa-file"></i></span>
-<span class="box-rounded node-count">{{ $ctrl._viewModel.flattenCount }}<i class="fa fa-eye-slash"></i></span>
-<span class="box-rounded node-count">{{ $ctrl._viewModel.excludeCount }}<i class="fa fa-ban"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.fileCount }}<i class="fa fa-file"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.flattenCount }}<i class="fa fa-eye-slash"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.excludeCount }}<i class="fa fa-ban"></i></span>
+<div id="sort-component">
+	<sorting-button-component id="id-sort-component-button"></sorting-button-component>
```

Comment:
I meant to change the id from `id-sort-component-button` to `sort-component-button`